control/control.go

```diff
@@ -80,6 +82,7 @@ type Controller struct { // TODO: ControlService
 	gatewayForwarder *controlgateway.GatewayForwarder
 	throttledGC      func()
 	gcmu             sync.Mutex
+	controlapi.UnimplementedControlServer
```

Comment:
While this is recommended in grpc docs, I don't see how it improves things for us. I'd rather want to get a compilation error if the interface is missing a function.

Reply:
Added `--go-grpc_opt=require_unimplemented_servers=false` to the generate options to remove the need for this. We can always change this in the future if it ends up becoming a problem.

I believe the intention is to ensure that semver is followed for changes to services so code doesn't break in compilation. This isn't a concern of ours though.